e2e/files.spec.ts

```diff
@@ -280,4 +335,210 @@ test.describe('Files', () => {
       })
     },
   )
+
+  testAuth(
+    'should create and delete a file',
+    async ({ userPage, browserName }) => {
+      test.fail(
+        browserName === 'webkit',
+        `Playwright is not preserving the File's lastModified time in webkit, so 
+        file upload fails because it seems like the file was modified during 
+        upload. See https://github.com/microsoft/playwright/issues/27452. Fix 
+        planned for Playwright v1.40.`,
+      )
```

Comment:
In webkit, Playwright uses the current time for the lastModified time, rather than preserving the value on the File. Test marked as failed until the fixed version is released, likely in mid-November.

Reply:
This webkit engine is just causing all sorts of problems!